warehouse/ingest-core/pom.xml

```diff
@@ -18,6 +18,11 @@
             <groupId>com.timgroup</groupId>
             <artifactId>java-statsd-client</artifactId>
         </dependency>
+        <dependency>
+            <groupId>datawave</groupId>
```

Comment:
The group id may be different now (gov.nsa.datawave?)

Reply:
resolve conflicts please

Reply:
Conflict resolved